pm/README.md

```diff
@@ -0,0 +1,37 @@
+# Probabilistic Micropayments (PM) Client
+
+The `pm` package contains a client implementation of the [Livepeer PM protocol](https://github.com/livepeer/wiki/blob/master/spec/streamflow/pm.md). The main areas of interest are:
+
+- A `Ticket` type which defines the ticket format that both senders and recipients accept.
```

Comment:
It might be a good idea to provide some extra context around this type as its fieldnames are used extensively in the RNG part of the README. WDYT?